src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp

```diff
@@ -54,6 +63,39 @@ tnsr::AA<DataType, SpatialDim, Frame> inverse_spacetime_metric(
     const tnsr::II<DataType, SpatialDim, Frame>&
         inverse_spatial_metric) noexcept;
 
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Compute shift from spacetime metric and inverse spatial metric.
+ *
+ * \details Computes
+ * \f{align}
+ *    N^i &= g^{ij} \psi_{jt}
```

Comment:
Please add a reference to an equation number where this is derived to this comment.

Reply:
Not sure where to find an equation number; I derived this in 1 line from the definition of the lower shift as the t,i component of the 4-metric.

Reply:
Ok, I can find an equation that is close enough to this... I will add.